website/docs/concepts/recorder_service/README.md

```diff
@@ -0,0 +1,159 @@
+view of the robot's location and movement.
+- `gps/relposned`: Provides GPS relative position in North, East, Down (N-E-D) coordinates,
+offering a detailed spatial orientation of the robot.
+
+### Filter Topics
+
+- `filter/health`: Delivers health metrics for the Filter service, including publishing frequency.
+- `filter/state`: Streams the filter state, which combines the robot's pose, orientation,
+filter state uncertainties, and filter convergence status.
+This topic is essential for understanding the robot's current state and its accuracy.
+
+---
+
+These topics ensure that users have a comprehensive set of data streams to choose from,
+depending on their specific needs and analysis requirements.
+
+## Recording Profile
+
+To initiate recording, a "Recording profile" must be provided as an argument.
+This profile is a JSON file detailing the subscriptions or topics to be recorded. Here's an example:
+
+```json
+{
+    "config": {
+        "name": "record_default",
+        "subscriptions": [
+            {
+            "uri": {
+                "path": "*",
+                "query": "service_name=canbus"
+            },
+            "every_n": 1
+            },
+            {
+            "uri": {
+                "path": "*",
+                "query": "service_name=gps"
+            },
+            "every_n": 1
+            },
+            {
+            "uri": {
+                "path": "/imu",
+                "query": "service_name=oak0"
+            },
+            "every_n": 1
+            },
+            {
+            "uri": {
+                "path": "*",
+                "query": "service_name=filter"
+            },
+            "every_n": 1
+            }
+        ]
+    }
+}
+```
+
+:::info INFO
+The ***** symbol in the `path` field indicates that all available topics of that specific
+subscription (service) should be recorded.
+:::
+
+## API
+
+Users can interact with the Recorder Service using the following commands:
+
+- `/start`: Initiates data recording.
+A valid **Recording profile" must be provided as an argument.
+- `/stop`: Halts the data recording process.
+
+## Data Storage
+
+All recorded data is saved as binary files on the disk at /mnt/data.
+The naming convention for these files captures the precise timestamp (to the microsecond)
+at the recording's start, the robot's name, and the file number in the sequence:
+
+```php
+<yyyy>_<mm>_<dd>_<hh>_<mn>_<ss>_<msmsms>_<robot_name>.0000.bin
+```
+
+For example:
+
+```php
+2023_01_12_16_17_52_134845_element-vegetable.0000.bin
+```
+
+## How to Use
+
+Data can be recorded through the Recorder App, or using the `EventsClient` class.
+
+For a step-by-step guide on using the Recorder App, please refer to this
+[example](https://amiga.farm-ng.com/docs/examples/import_log_file/).
+
+For a step-by-step guide on recording data using the `EventsClient`, please refer to this
+[example](https://amiga.farm-ng.com/docs/examples/events_recorder/).
```

Comment:
@guilhermedemouraa prefer (1) relative docs links if it is to another page in our docs and (2) describing the page the link goes to instead of generic 'here', 'example' 'this page' , etc.
```suggestion
For a step-by-step guide on using the Recorder App, please refer to the
[Record and Access Data example](/docs/examples/import_log_file/).

For a step-by-step guide on recording data using the `EventsClient`, please refer to the
[Events Recorder example](/docs/examples/events_recorder/).
```

Reply:
I added this in https://github.com/farm-ng/amiga-dev-kit/pull/226/commits/eef0c7a3a3163a7a88c873b93da2aeacec376549, but wanted to give visibility